optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -0,0 +1,670 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2017
+# Author: Cleber Rosa <crosa@redhat.com>
```

Comment:
About the authorship, it wasn't really intentional.  I started creating a new file, and them moving content.  The original headers weren't moved.  I will fix it.

As a side note, authorship can always ve verified on the project history.

About the module size, we're talking about 670 lines of code.  If we compare that with the other modules we have (in `core` alone):
```
    13 avocado/core/__init__.py
    24 avocado/core/version.py
    30 avocado/core/job_id.py
    42 avocado/core/exit_codes.py
    58 avocado/core/status.py
    86 avocado/core/app.py
    91 avocado/core/decorators.py
   114 avocado/core/result.py
   147 avocado/core/parser.py
   162 avocado/core/mux.py
   169 avocado/core/safeloader.py
   179 avocado/core/exceptions.py
   198 avocado/core/dispatcher.py
   202 avocado/core/jobdata.py
   203 avocado/core/plugin_interfaces.py
   260 avocado/core/data_dir.py
   267 avocado/core/settings.py
   488 avocado/core/varianter.py
   499 avocado/core/tree.py
   562 avocado/core/runner.py
   587 avocado/core/job.py
   635 avocado/core/sysinfo.py
   709 avocado/core/output.py
   862 avocado/core/loader.py
   919 avocado/core/test.py
```

We have others which are larger.  Do you see the module breakdown as a blocker here?